src/configs/directory.rs

```diff
@@ -22,6 +24,15 @@ impl<'a> RootModuleConfig<'a> for DirectoryConfig<'a> {
             fish_style_pwd_dir_length: 0,
             use_logical_path: true,
             prefix: "in ",
+            separator: match std::path::MAIN_SEPARATOR {
+                '/' => "/",
+                '\\' => "\\",
+                '.' => ".",
+                ':' => ":",
+                '>' => ">",
+                _ => panic!("Unknown Seperator"),
```

Comment:
**issue**:
I don't think it would be appropriate for the prompt to panic if the `MAIN_SEPARATOR` isn't on this list. Perhaps we should fallback on `/`?

Reply:
I didn't suggest that so that any such emerging problems can be handled quickly. Fallback sounds good too.

Reply:
Your call guys. Either paths are Ok to me.